web/vtadmin/src/api/http.ts

```diff
@@ -620,22 +620,6 @@ export const workflowDelete = async ({ clusterID, request }: WorkflowDeleteParam
     return vtctldata.WorkflowDeleteResponse.create(result);
 };
 
-export const fetchVTExplain = async <R extends pb.IVTExplainRequest>({ cluster, keyspace, sql }: R) => {
-    // As an easy enhancement for later, we can also validate the request parameters on the front-end
-    // instead of defaulting to '', to save a round trip.
-    const req = new URLSearchParams();
-    req.append('cluster', cluster || '');
-    req.append('keyspace', keyspace || '');
-    req.append('sql', sql || '');
-
-    const { result } = await vtfetch(`/api/vtexplain?${req}`);
-
-    const err = pb.VTExplainResponse.verify(result);
-    if (err) throw Error(err);
-
-    return pb.VTExplainResponse.create(result);
-};
-
 export const fetchVExplain = async <R extends pb.IVExplainRequest>({ cluster_id, keyspace, sql }: R) => {
     // As an easy enhancement for later, we can also validate the request parameters on the front-end
     // instead of defaulting to '', to save a round trip.
```

Comment:
not related to this PR, but I think we can remove this comment since we already added validation for this in the earlier PR.